loopd/config.go

```diff
@@ -99,9 +99,8 @@ var (
 	)
 
 	// DefaultAutogenValidity is the default validity of a self-signed
-	// certificate. The value corresponds to 14 months
-	// (14 months * 30 days * 24 hours).
-	DefaultAutogenValidity = 14 * 30 * 24 * time.Hour
+	// certificate in number of days. The value corresponds to 420 days (~14 months)
+	DefaultAutogenValidity = 420
```

Comment:
Why not align this with 1 year i.e `365` ?

Reply:
I am using current default here. Happy to change that to 365